Config/dev_env/interfaces.ts

```diff
@@ -0,0 +1,22 @@
+import { storedGutka, entryObj, GutkaApi, GlobalApi, SearchApi, ViewerApi } from "./types";
+export interface IGutkaCtx {
+  gutkaApi: GutkaApi
```

Comment:
replace this with the functions and state provided so that we can get vscode intellisense